doc/recognition.xml

```diff
@@ -352,8 +352,17 @@ set this to <K>false</K>. The result is either <K>fail</K> or
 <K>false</K> if the method fails or a recognition info record <C>ri</C>.
 If the method created a leaf then <C>ri</C> will be a leaf, otherwise
 it will have the attribute <Ref Attr="Homom"/> set, but no factor or
-kernel have been created or recognised yet. You can use for example
-the methods in <Ref Var="FindHomMethodsPerm"/> or
+kernel have been created or recognised yet.
+
+<P/>GAP homomorphisms are not required to give a sensible answer when
+given a value not in their source, and in practice often enter the break
+loop, or return an incorrect answer. This causes problems when checking
+if a value is not in the represented group. To avoid this problem,
+<Ref Attr="validatehomominput"/> can be assigned to a function.
```

Comment:
This seems to be phrased the wrong way around: not `validatehomominput` is assigned to a function, but rather a function is assigned to `validatehomominput`; or `validatehomominput` is set to a function.
```suggestion
<Ref Attr="validatehomominput"/> can be set to a function.
```